CONTRIBUTING.md

```diff
@@ -0,0 +1,339 @@
+---
+
+[![flyve hero](/images/flyve-mdm-banner.png)](https://flyve-mdm.com/)
+
+# How to contribute to Flyve MDM GLPI Plugin
+
+Welcome to our ever-growing community :octocat:!
+
+We are more than happy to accept external contributions to the project in the form of feedback, translations, bug reports and even better, pull requests.
+
+We present you the guidelines to start contributing in any of the Flyve MDM projects.
+
+# Table of contents:
+
+- 1 [See what’s going on](#1)
+    - 1.1 [Issue Dashboard](#1.1)
+    - 1.2 [Pull Request Dashboard](#1.2)
+- 2 [Assistance](#2)
+    - 2.1 [Live Support](#2.1)
+    - 2.2 [Technical Questions](#2.2)
+    - 2.3 [Discussion](#2.3)
+    - 2.4 [Customers Assistance](#2.4)
+- 3 [Feature Requests](#3)
+    - 3.1 [Requirement for a Feature Request](#3.1)
+        - 3.1.1 [Major Feature Request](#3.1.1)
+        - 3.1.2 [Minor Feature Request](#3.1.2)
+    - 3.2 [Request a New Feature](#3.2)
+    - 3.3 [Implement a New Feature](#3.3)
+- 4 [Submitting](#4)
+    - 4.1 [How to Submit an Issue](#4.1)
+        - 4.1.1 [Check for Past Issues](#4.1.1)
+        - 4.1.2 [Information Needed](#4.1.2)
+        - 4.1.3 [Submit an Issue](#4.1.3)
+    - 4.2 [How to Create a Pull Request (PR)](#4.2)
+        - 4.2.1 [Create a Branch](#4.2.1)
+        - 4.2.2 [Make Changes](#4.2.2)
+        - 4.2.3 [Commit Your Changes](#4.2.3)
+            - 4.2.3.1 [Rules to Follow](#4.2.3.1)
+            - 4.2.3.2 [Commit Format](#4.2.3.2)
+                - 4.2.3.2.1 [Header: Writting a `<type>`](#4.2.3.2.1)
+                - 4.2.3.2.2 [Header: Writting the (`<optional scope>`)](#4.2.3.2.2 )
+                - 4.2.3.2.3 [Header: Writting a `<description>`](#4.2.3.2.3)
+                - 4.2.3.2.4 [Header Lenght](#4.2.3.2.4)
+                - 4.2.3.2.5 [Writting the `<optional body>`](#4.2.3.2.5)
+                - 4.2.3.2.6 [Writting the `<optional footer>`](#4.2.3.2.6)
+            - 4.2.3.3 [Commit Examples](#4.2.3.3)
+        - 4.2.4 [Push your Changes](#4.2.4)
+        - 4.2.5 [Create a Pull Request](#4.2.5)
+            - 4.2.5.1 [How to Write a Title for a Pull Request.](#4.2.5.1)
+            - 4.2.5.2 [How to Write a Description for a Pull Request.](#4.2.5.2)
+            - 4.2.5.1 [Additional Information.](#4.2.5.1)
+        - 4.2.6 [How to proceed with suggestions.](#4.2.6)
+- 5 [What to do next?](#5)
+- 6 [Coding Rules](#6)
+
+# <a name="1"></a> 1. See what's going on!
+
+
+## <a name="1.1"></a> 1.1 Issue Dashboard :fire:
+If you want to know all the issues we're dealing with right now, take a look at our [Issue Dashboard](https://github.com/flyve-mdm/glpi-plugin/issues) and look for areas in which you can help.
+
+
+## <a name="1.2"></a> 1.2 Pull Request Dashboard :fire_engine:
+If you want to give us a hand solving issues then great, take a look at our [Pull Request Dashboard](https://github.com/flyve-mdm/glpi-plugin/issues) and check for an open or closed PR. We don’t want to duplicate efforts.
```

Comment:
Hi, @bootbeep 

This link takes you to the issue dashboard

Reply:
Thanks!